src/pages/EditOrganization.tsx

```diff
@@ -121,13 +121,20 @@ const UsersTable = () => {
     .catch(() => history.push('/dashboard'));
   }, []);
 
+  const onClickRemoveUser = (removedUser:OrganizationUser) =>
+  {
+    OrganizationService.deleteOrganizationUser(removedUser.organizationId,removedUser.userId);
+    window.location.reload();
```

Comment:
It may be better to just remove the deleted user from the list than reload the whole page.